.pre-commit-config.yaml

```diff
@@ -2,17 +2,26 @@ repos:
   - repo: local
     hooks:
       - id: black
+        stages: [commit]
         name: black
         language: system
         entry: black
         types: [python]
       - id: isort
+        stages: [commit]
         name: isort
         language: system
-        entry: isort -y
+        entry: isort --settings-path api/setup.cfg
```

Comment:
I suggest doing `isort` before `black`, as it sometimes generates import statements that black wouldn't accept, causing `flake8` to trip

Reply:
Done. In all our other projects, we have black before isort...

Reply:
Hmm, maybe we should change that some day. I lost a few hours once debugging my editor, because it ran black before isort, and the generated code would then not be accepted. Works in most cases though, so it's not *that* urgent

Reply:
Days since my last fight with isort: 1
:rofl: